Project/js/filter.js

```diff
@@ -0,0 +1,15 @@
+const list = document.getElementsByClassName("cities-list__element");
+const input = document.querySelector(".content-search__field");
+
+function search() {
+    for (let i = 0; i < list.length; i++) {
+        if (list[i].querySelector(".city-box__title").innerHTML.toLowerCase().indexOf(input.value.toLowerCase()) > -1) {
```

Comment:
please store `input.value.toLowerCase()` in a variable before the loop, so that it's not executed N times